src/libs/actions/Report.js

```diff
@@ -795,6 +795,46 @@ function addAction(reportID, text, file) {
         .then(({reportAction}) => updateReportWithNewAction(reportID, reportAction));
 }
 
+/**
+ * Deletes a comment from the report, basically sets it as empty string
+ *
+ * @param {Number} reportID
+ * @param {Object} reportAction
+ */
+function deleteReportAction(reportID, reportAction) {
```

Comment:
Also, depending on whether Edit Comments is finished first, I think it might be easier to just have this function do:

```javascript
function deleteReportAction(reportID, reportAction) {
    editReportComment(reportID, reportAction, '');
}
```

Reply:
Also, I think we should rename this to `deleteReportComment` to closer match the API endpoint.

Reply:
at the moment edit comment is not yet done, should I wait?